src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAGenerateRuntimeConfigurationFiles.cs

```diff
@@ -89,21 +90,21 @@ public void Given3RuntimeFrameworksItCanGenerateWithoutAssetFile()
                         "Microsoft.NETCore.App",
                         new Dictionary<string, string>
                         {
-                            {"FrameworkName", "Microsoft.NETCore.App"}, {"Version", "3.1.0"}
+                            {"FrameworkName", "Microsoft.NETCore.App"}, {"Version", "6.0.5"}
```

Comment:
It probably makers sense to make all of these 6.0.0 versions. See how before it was 3.1.0. I think it's simpler if we're specifying a specific version of 6.0 to go with the first one as otherwise, different tests might end up doing different ones and anyone reading the code might wonder why we picked .5.